lms/static/sass/xmodule/_headings.scss

```diff
@@ -88,29 +88,33 @@ $headings-base-color:                       $gray-d2;
     letter-spacing: 1px;
 }
 
-.xblock .xblock {
+// let's make these classnames available to the entire LMS!
+
+.hd-1,
```

Comment:
Ah yes, the `xmodule` directory is problematic. How about a `_headings.scss` partial in the main directory (alongside `_base.scss`, and the other extends and mixins)?